hikari/impl/stateful_event_manager.py

```diff
@@ -28,6 +28,7 @@
 import asyncio
 import typing
 
+from hikari import Snowflake
```

Comment:
For lib consistency we never want to import classes directly, we'd likely do (from hikari import snowflakes) then access snowflakes.Snowflake. Also did `nox -s isort` really order these invites like this?